command/certificate/extractPE.go

```diff
@@ -0,0 +1,168 @@
+package certificate
+
+import (
+	"debug/pe"
+	"encoding/asn1"
+	"encoding/binary"
+	"encoding/pem"
+	"github.com/pkg/errors"
+	"github.com/smallstep/cli/errs"
+	"github.com/urfave/cli"
+	"go.mozilla.org/pkcs7"
+	"io"
+	"os"
+	"path"
+)
+
+var ErrCodeSignCertsNoCertificateTableFound = errors.New("No Certificate Table found")
+
+func extractPECommand() cli.Command {
+	return cli.Command{
+		Name:        "extract-pe",
+		Action:      cli.ActionFunc(extractPEAction),
+		Usage:       `TODO`,
+		UsageText:   `TODO`,
+		Description: `TODO`,
+	}
+}
+
+func extractPEAction(ctx *cli.Context) error {
+	if err := errs.MinMaxNumberOfArguments(ctx, 1, 1); err != nil {
+		return err
+	}
+
+	file, err := os.Open(path.Clean(ctx.Args().Get(0)))
+	if err != nil {
+		return err
```

Comment:
Wrap errors coming from the stdlib, they allow us to see an stack trace of the error using the environment variable STEPDEBUG. Do something like:
```go
return errors.Wrapf(err, "error opening %s", filename)
```

Reply:
Try to return always errors from `github.com/pkg/errors`.